source/renderer/app/components/wallet/settings/WalletRecoveryPhraseVerificationWidget.js

```diff
@@ -235,16 +233,7 @@ export default class WalletRecoveryPhraseVerificationWidget extends Component<Pr
       <div className={styles.component}>
         <h2>{intl.formatMessage(messages.title)}</h2>
         <div className={styles.description}>
-          {intl.formatMessage(messages.description, {
```

Comment:
@daniloprates why did we remove this "paper wallet" tooltip part?

Reply:
Missed that when re-enabling the Byron bit. Done, now.